app/models/spree/order.rb

```diff
@@ -395,7 +395,8 @@ def finalize!
     def deliver_order_confirmation_email
       return if subscription.present?
 
-      ConfirmOrderJob.perform_later(id)
+      Spree::OrderMailer.confirm_email_for_customer(id).deliver_later
+      Spree::OrderMailer.confirm_email_for_shop(id).deliver_later
```

Comment:
The specs say that we don't have `deliver_later`. Maybe we need to configure that first? We have just been using the Delayed::Job API until now.